test/t8n/t8n.cpp

```diff
@@ -241,7 +242,38 @@ int main(int argc, const char* argv[])
         if (rev >= EVMC_PRAGUE)
         {
             // EIP-7685: General purpose execution layer requests
-            j_result["requestsRoot"] = hex0x(state::EMPTY_MPT_HASH);
+            j_result["requests"] = json::json::array();
+            // TODO: actual requests should be used in the following lines, for now all empty.
+            const uint8_t withdrawals[1] = {0x00};
```

Comment:
:see_no_evil: This is horrible, but let me know if this is good enough for the time being, or if there are any low-hanging cleanups that can be made here. I don't want to overthink it unnecessarily

Reply:
actually, having typed this, I got a thought that I could just hard code the empty hash, without all the sha256 work.